plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -3684,8 +3715,11 @@ def collect_webhook_notification_playbook_params(self, webhook_notification_deta
         if sites:
             site_ids = self.get_site_ids(sites)
             if not site_ids:
-                self.msg = "Unable to find the Site IDs for the given site(s) - '{0}' in the playbook's input.".format(sites)
-                self.log(self.msg, "INFO")
+                site_msg = (
+                    "Unable to find the Site IDs for the given site(s) - '{0}' in the playbook's input for the "
```

Comment:
"No Site IDs were found for the specified site(s) - '{0}' in the playbook input during the "
    "Webhook event notification operation."

Reply:
updated.